nats/src/main/java/io/micronaut/nats/annotation/NatsListener.java

```diff
@@ -49,4 +49,12 @@
      */
     String queue() default "";
 
+    /**
+     * Sets the client id of the Nats consumer. If not specified the client id is configured
+     * to be the value of {@link io.micronaut.runtime.ApplicationConfiguration#getName()}.
+     *
+     * @return The client id
+     */
+    String clientId() default "";
```

Comment:
can you add `@since` with the next version this will go into

Reply:
done